scripts/README.md

```diff
@@ -0,0 +1,15 @@
+## Scripts
+This directory contains various scripts to help you set-up and analyze results
+of your SaltProc simulations.
+
+### `xsdata`
```

Comment:
Should you add in a note about changing the temperature variables for the cross section data?

Reply:
Great suggestion. I'll add this.